dist/APIManager.js

```diff
@@ -0,0 +1,77 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+
+    constructor(renderer)
+     {
+         this.cityData = []
+         this.renderer = renderer
```

Comment:
You should not use your renderer in here. Try to make your code MVC. The main.js talks to the manager, then the manager sends back the data, and then the main.js sends that data to the renderer. The Manager should not have anything to do with the renderer.

Reply:
okay